### ISSUE TITLE
2.13 crash

### ISSUE THREAD

**Issue by Djfe** [RandomEngy/VidCoder#6]
```
`System.Exception: An OnError occurred on an object (usually ObservableAsPropertyHelper) that would break a binding or command. To prevent this, Subscribe to the ThrownExceptions property of your objects ---> System.EntryPointNotFoundException: Der Einstiegspunkt "hb_check_h264_level" wurde nicht in der DLL "hb.dll" gefunden.
   bei HandBrake.ApplicationServices.Interop.HbLib.HBFunctions.hb_check_h264_level(String level, Int32 width, Int32 height, Int32 fps_num, Int32 fps_den, Int32 interlaced, Int32 fake_interlaced)
   bei HandBrake.ApplicationServices.Interop.HandBrakeUtils.IsH264LevelValid(String level, Int32 width, Int32 height, Int32 fpsNumerator, Int32 fpsDenominator, Boolean interlaced, Boolean fakeInterlaced) in D:\docs\projects\HandBrakeGit\win\CS\HandBrake.ApplicationServices\Interop\HandBrakeUtils.cs:Zeile 225.
   bei VidCoder.ViewModel.LevelChoiceViewModel.<RegisterIsCompatible>b__5_0(Boolean hasVideoSource, FrameRate titleFramerate, Double profileFramerate, String videoOptions, Geometry outputSize) in D:\docs\projects\VidCoder\VidCoder\ViewModel\DataModels\LevelChoiceViewModel.cs:Zeile 73.
   bei System.Reactive.Linq.ObservableImpl.CombineLatest`6._.GetResult()
   bei System.Reactive.Linq.ObservableImpl.CombineLatestSink`1.Next(Int32 index)
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei ReactiveUI.RxApp.<>c__DisplayClass5.<.cctor>b__2()
   bei System.Reactive.Concurrency.Scheduler.Invoke(IScheduler scheduler, Action action)
   bei System.Reactive.Concurrency.DispatcherScheduler.<>c__DisplayClass1`1.<Schedule>b__0()
   bei System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   bei System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```
probably not very helpful for you
since I can only post one of the crash windows' content
I was able to copy the content of one window and then press ok, but then VidCoder crashed
even though there were at least 10 further error windows :O
I couldn't find them in the appdata folder -> VidCoder should write such errors if possible to the appdata folder too (in addition to encode logs)

and it shouldn't crash if there are several error windows

anyways, I was quite surprised
I simply opened the beta for the first time, created a new preset based on the high preset and opened a movie and then suddenly 10 error windows opened

**Comment by Djfe** [comment]
just to make it clear: I wasn't able to reproduce the errors by opening the same or another file

**Comment by RandomEngy** [comment]
Fixed in 2.14 Beta.

**Comment by Djfe** [comment]
you found out what caused the error? (of all error windows)
or just what caused the error of the log I posted above?

**Comment by RandomEngy** [comment]
Well I thought I did... apparently they removed the H264 level checker method. Trying to find out why.

**Comment by RandomEngy** [comment]
Fixed in 2.15 Beta by removing the ability to check valid H.264 levels. Will re-enable this functionality when j45 adds it back in to HandBrake.
